src/components/CohortBuilder/Summary/Cards/StudiesChart.js

```diff
@@ -138,13 +160,15 @@ class StudiesChart extends React.Component {
           <HorizontalBar
             showCursor={true}
             data={data}
+            tooltipDictionary={studies.map((s) => ({ label: s.code, tooltip: s.name }))}
```

Comment:
can it be transformed at the source? In the `transform` func for instance.

Reply:
yup.